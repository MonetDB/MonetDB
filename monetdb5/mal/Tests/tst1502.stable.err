stderr of test 'tst1502` in directory 'monetdb5/mal` itself:
