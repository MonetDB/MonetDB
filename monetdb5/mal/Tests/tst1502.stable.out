stdout of test 'tst1502` in directory 'monetdb5/mal` itself:


# 15:36:14 >  
# 15:36:14 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33827" "--set" "mapi_usock=/var/tmp/mtest-16148/.s.monetdb.33827" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/export/scratch1/mk/default//Linux/var/MonetDB/mTests_monetdb5_mal" "--set" "mal_listing=2" "--set" "embedded_r=yes"
# 15:36:14 >  

# MonetDB 5 server v11.22.0
# This is an unreleased version
# Serving database 'mTests_monetdb5_mal', using 8 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs and 128bit integers dynamically linked
# Found 15.589 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://vienna.da.cwi.nl:33827/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-16148/.s.monetdb.33827
# MonetDB/GIS module loaded
# Start processing logs sql/sql_logs version 52200
# Start reading the write-ahead log 'sql_logs/sql/log.4'
# Finished reading the write-ahead log 'sql_logs/sql/log.4'
# Finished processing logs sql/sql_logs
# MonetDB/SQL module loaded
# MonetDB/R   module loaded


# 15:36:15 >  
# 15:36:15 >  "mclient" "-lmal" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-16148" "--port=33827"
# 15:36:15 >  

#	io.print("sort x"); 
[ "sort x" ]
#--------------------------#
# h	t  # name
# void	bte  # type
#--------------------------#
[ 0@0,	1  ]
[ 1@0,	  1	  ]
[ 2@0,	2  ]
[ 3@0,	  2	  ]
[ 4@0,	  2	  ]
[ 5@0,	  2	  ]
[ 6@0,	3  ]
[ 7@0,	  3	  ]
[ "hsort xrev" ]
#--------------------------#
# h	t  # name
# void	bte  # type
#--------------------------#
[ 0@0,	3  ]
[ 1@0,	3  ]
[ 2@0,	2  ]
[ 3@0,	2  ]
[ 4@0,	2  ]
[ 5@0,	2  ]
[ 6@0,	1  ]
[ 7@0,	1  ]


